core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -1071,6 +1071,24 @@ Map<String,Integer> listConstraints(String tableName)
   List<DiskUsage> getDiskUsage(Set<String> tables)
       throws AccumuloException, AccumuloSecurityException, TableNotFoundException;
 
+  /**
+   * Gets the number of bytes being used in the files for a set of tables. This operation will use
+   * the client to scan the metadata table to compute the size metrics for the tables. For the most
+   * accurate information a flush or compaction can be run first on the set of tables.
```

Comment:
Should note that the sites in the metadata table are estimates.

Reply:
And importantly, not all the files have these estimated file sizes. They aren't required.